nav2_dwb_controller/dwb_plugins/src/standard_traj_generator.cpp

```diff
@@ -156,9 +161,14 @@ dwb_msgs::msg::Trajectory2D StandardTrajectoryGenerator::generateTrajectory(
   double running_time = 0.0;
   std::vector<double> steps = getTimeSteps(cmd_vel);
   traj.poses.push_back(start_pose);
+  bool firstVel = false;
```

Comment:
Variables should be `snake_case` not `camelCase`

Reply:
Updated this, thanks for pointing it out!